docs/getting-started.md

```diff
@@ -215,6 +215,7 @@ for some languages:
 - [Google Go](https://github.com/ldmberman/GoEV3)
 - [Node.js](https://github.com/WasabiFan/ev3dev-NodeJS)
 - [Python](https://github.com/topikachu/python-ev3)
+- [C](https://github.com/theZiz/ev3c)
```

Comment:
I think it should be replaced with a link as well -- this list is pretty outdated.